test/uk/gov/hmrc/cipphonenumberverification/connectors/CircuitBreakersSpec.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cipphonenumberverification.connectors
+
+import akka.stream.Materializer
+import com.github.tomakehurst.wiremock.client.WireMock.{aResponse, get, post, stubFor, urlEqualTo}
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import play.api.http.Status.OK
+import play.api.libs.json.Json
+import play.api.libs.ws.ahc.AhcCurlRequestLogger
+import play.api.test.Helpers.{await, defaultAwaitTimeout}
+import uk.gov.hmrc.cipphonenumberverification.config.CircuitBreakerConfig
+import uk.gov.hmrc.cipphonenumberverification.utils.TestActorSystem
+import uk.gov.hmrc.http.HttpReadsInstances.readEitherOf
+import uk.gov.hmrc.http.test.{HttpClientV2Support, WireMockSupport}
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse, StringContextOps, UpstreamErrorResponse}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.Duration
+import scala.util.{Failure, Success, Try}
+
+class CircuitBreakersSpec extends AnyWordSpec
```

Comment:
singular name?

Reply:
Left as that because inside the trait we are using a CircuitBreaker instance